engine/src/java/com/runrev/android/libraries/LibBrowser.java

```diff
@@ -321,10 +321,14 @@ public void onHideCustomView()
 					m_custom_view_callback = null;
 				}
 			}
+			public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermissions.Callback callback) {
+				callback.invoke(origin, true, false);
```

Comment:
@livecodepanos I'm thinking we should be presenting a dialog here

Reply:
Sure, I'll look into adding one